internal/builders/go/pkg/config.go

```diff
@@ -158,7 +164,7 @@ func convertPathError(e error, msg string) error {
 func validateVersion(cf *goReleaserConfigFile) error {
 	_, exists := supportedVersions[cf.Version]
 	if !exists {
-		return fmt.Errorf("%w:%d", ErrorUnsupportedVersion, cf.Version)
+		return errors.Errorf(&ErrUnsupportedVersion{}, "version '%d' not supported", cf.Version)
```

Comment:
I'm not super familiar with wrappable error. But how do we make sure the error message remains consistent across the codebase?